e2e/src/utils/retries.js

```diff
@@ -1,14 +1,20 @@
+import { merge } from 'lodash'
 import { retry } from 'ts-retry-promise'
 
-export const launchApp = async (
-  launchArgs = {
-    newInstance: true,
-    permissions: { notifications: 'YES', contacts: 'YES', camera: 'YES' },
-    launchArgs: {
-      detoxPrintBusyIdleResources: 'YES',
-    },
-  }
-) => {
+const defaultLaunchArgs = {
+  newInstance: true,
+  permissions: { notifications: 'YES', contacts: 'YES', camera: 'YES' },
+  launchArgs: {
+    detoxPrintBusyIdleResources: 'YES',
+    // Use new tx feed from Zerion by default
+    statsigGateOverrides: 'show_zerion_transaction_feed=true',
```

Comment:
do any tests set statsigGateOverrides currently? I think no, but if we eventually want to do it, we'd have to remember to also include `show_zerion_transaction_feed=true` in those tests right? Would be nice if there's a way we can avoid that (e.g., take this as a separate object arg in `launchApp` and then construct the string format here), could just do this in a follow up or when we require other overrides

Reply:
I don't see a problem with punting on this until other e2e tests need to use `statsigGateOverrides`.